src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    public String getRandomColor() {
+        String[] colors = {"white", "black", "orange", "yellow", "grey", "blue"};
+        int randomIndex = new Random().nextInt(colors.length);
+        return colors[randomIndex];
```

Comment:
Consider defining the 'Random' object as a class-level variable instead of creating a new instance in each call to 'getRandomColor'. This is more efficient, as it reuses the same Random instance rather than creating a new one each time.